test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -2152,3 +2156,151 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator, FromEn
         EXPECT_TRUE(static_cast<const PositionDerivative&>(*propagator.getDynamics()[4]).isDefined());
     }
 }
+
+class OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator_Data_Success
+    : public OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator,
+      public ::testing::WithParamInterface<Tuple<Instant, Instant, Duration, String>>
+{
+};
+
+INSTANTIATE_TEST_SUITE_P(
+    Input_Data_Spans_Valid,
+    OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator_Data_Success,
+    testing::Values(
+        std::make_tuple(
+            Instant::Now() - Duration::Weeks(52 * 5),
+            Instant::Now() - Duration::Weeks(52 * 4.9),
+            Duration::Days(7),
+            "NRLMSISE Past Data limit pass"
+        ),
+        std::make_tuple(
+            Instant::Now() + Duration::Weeks(52 * 0.9),
+            Instant::Now() + Duration::Weeks(52 * 1.0),
+            Duration::Days(7),
+            "IERS Finals2000A Future Data limit pass"
```

Comment:
Is this parameter being used?

Reply:
It's not, but I thought it was useful to describe the purpose of the test case. If it grosses you out, I can make it a comment 😄

Reply:
Fair point. So you can actually make it print out a name for the test like so:
```
INSTANTIATE_TEST_SUITE_P(
    MyInstantiation, MyTestSuite,
    testing::Values(...),
    [](const testing::TestParamInfo<MyTestSuite::ParamType>& info) {
      return info.param[3];
    });
```
Something like that.